pkg/clustermanager/manager.go

```diff
@@ -315,6 +317,15 @@ func ToRESTConfig(cluster *v3.Cluster, context *config.ScaledContext) (*rest.Con
 					ht.DialTLS = tlsDialer
 				}
 			}
+			if cluster.Status.Driver == "googleKubernetesEngine" {
+				ts, err := gke.GetTokenSource(context.RunContext, (*cluster.Spec.GenericEngineConfig)["credential"].(string))
+				if err != nil {
```

Comment:
why don't we return the error here? should we log the error?

Reply:
Will log the error.